packages/platform-android/src/commands/runAndroid/adb.ts

```diff
@@ -34,6 +34,11 @@ function parseDevicesResult(result: string): Array<string> {
  */
 function getDevices(adbPath: string): Array<string> {
   try {
+    //If the path includes spaces (for example, the name of the USER_NAME) in Windows...
+    if (adbPath.includes(' ') && process.platform.startsWith('win')) {
+      adbPath = `"${adbPath}"`;
```

Comment:
Does this conditional matter? Can't we always wrap it with " " for the safety? I think that would work on Mac too.

Reply:
I don't have a Mac to test it, but I will find one.
The conditional is because I don't want break any thing.